test/BTCUtils.test.js

```diff
@@ -1,7 +1,7 @@
 const assert = require('assert');
 const ganache = require('ganache-cli');
 const Web3 = require('web3');
-const web3 = new Web3(ganache.provider());
+const web3 = new Web3(ganache.provider(), null, { transactionConfirmationBlocks: 1 });
 const compiledBTCUtils = require('../build/BTCUtils.json');
 const compiledBytes = require('../build/BytesLib.json');
```

Comment:
Prefer `compiledBytesLib` :)

Reply:
I didn't want to change too much in the original implementation in this PR. It contains only changes necessary for solc update.

Reply:
I agree with @nkuba's strategy here. Let's keep this PR minimal and change what really needs to be changed for the upgrade. Every project has its own naming conventions.

Reply:
^^ sorry didn't see that